src/mmw/js/src/account/views.js

```diff
@@ -13,7 +15,63 @@ var Clipboard = require('clipboard'),
     accountTmpl = require('./templates/account.html');
 
 var ProfileView = Marionette.ItemView.extend({
-    template: profileTmpl
+    template: profileTmpl,
+
+    ui: {
+        firstName: '#first_name',
+        lastName: '#last_name',
+        organization: '#organization',
+        userType: '#user_type',
+        country: '#country',
+        postalCode: '#postal_code',
+        saveChanges: '[data-action="save_changes"]'
+    },
+
+    events: {
+        'click @ui.saveChanges': 'saveChanges'
+    },
+
+    initialize: function(){
+        this.listenTo(this.model, 'change', this.render);
```

Comment:
This can be replaced with the Marionette [`modelEvents`](https://marionettejs.com/docs/v2.4.1/marionette.view.html#viewmodelevents-and-viewcollectionevents) hash, like so:

```js
events: {
    'click @ui.saveChanges': 'saveChanges',
},

modelEvents: {
    'change': 'render',
},
```

Reply:
Addressed in fixup 2259d26